Modules/Registration/PDEDeformable/wrapping/itkLevelSetMotionRegistrationFilter.wrap

```diff
@@ -1,7 +1,18 @@
-# DemonsRegistrationFilter is wrapped with POINTER_WITH_SUPERCLASS
+# LevelSetMotionRegistrationFilter is wrapped with POINTER_WITH_SUPERCLASS
```

Comment:
I guess that comment was referring to this class' superclass? This change makes the comment factually incorrect, as this file wraps LevelSetMotionRegistrationFilter as POINTER on line 3.